src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/BowlRitualRecipe.kt

```diff
@@ -61,14 +66,14 @@ class BowlRitualRecipe(
         // Remove all ingredients
         tile.clear()
 
-        applyEffects(tile.world!!, tile.pos, tile)
+        applyEffects(tile.world!!, tile.pos, tile, tile.specialItem)
```

Comment:
Maybe it's better to create a `PlaySoundEffect`, so that we can change the visual and audible feedback per ritual? Also in regard of future `negative_effects`.

Also for the `PoitionEffect` the visuals are part of the effect.

See also upper comment.

Reply:
See #49 for follow up